modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -28,7 +35,7 @@
                   :disabled="disabled"
                   type="primary"
                   label="Login"
-                  buttonType="submit"
+                  button-type="submit"
```

Comment:
lol

Reply:
That was auto-corrected from our eslint config.